wpilibc/src/main/native/include/frc/BuiltInAccelerometer.h

```diff
@@ -7,19 +7,18 @@
 #include <wpi/sendable/Sendable.h>
 #include <wpi/sendable/SendableHelper.h>
 
-#include "frc/interfaces/Accelerometer.h"
-
 namespace frc {
 
 /**
  * Built-in accelerometer.
  *
  * This class allows access to the roboRIO's internal accelerometer.
  */
-class BuiltInAccelerometer : public Accelerometer,
-                             public wpi::Sendable,
+class BuiltInAccelerometer : public wpi::Sendable,
                              public wpi::SendableHelper<BuiltInAccelerometer> {
  public:
+  enum Range { kRange_2G = 0, kRange_4G = 1, kRange_8G = 2, kRange_16G = 3 };
```

Comment:
Since each accelerometer has their own range now, we should limit the ranges to what are actually supported
```suggestion
  enum Range { kRange_2G = 0, kRange_4G = 1, kRange_8G = 2 };
```

Reply:
I don't know what those are supposed to be.